todo_app/data/database_items.py

```diff
@@ -0,0 +1,51 @@
+from flask import session
+import os
+import pymongo
+from bson import ObjectId
+
+def get_cards():
+    client = pymongo.MongoClient(os.environ.get('CONNECTIONSTRING'))
+    db = client['test-database']
+    collection = db.test_collection
+    items=[]
+    returned_items=[]
+    for document in collection.find():
+            items.append(document)
+    
+
+    for item in items:
+        my_item=Item(item["_id"],item["name"],item["desc"],item["due"],item["status"])
+        returned_items.append(my_item)
```

Comment:
Here you loop over everything twice; not terrible but if this gets slow (e.g. if we have a large DB later) then combining these loops could cut our loading time by half! Currently the first loop basically constructs a list of the same items that `collection.find()` returns, so skipping that lets us simplify (untested!) this to:
```suggestion
    returned_items=[]
    for item in collection.find():
        my_item=Item(item["_id"],item["name"],item["desc"],item["due"],item["status"])
        returned_items.append(my_item)
```